SETUP.md

```diff
@@ -2,42 +2,14 @@
 
 This guide is to get you setup with the tools you'll need for development.
 
-**Date**: May 24, 2002
-
-**Platform**: Fedora 35, 64-bit
-
-1. Install _SDKMan_, [link](https://sdkman.io/install).
-   1. `curl -s "https://get.sdkman.io" | bash`
-   2. `source "$HOME/.sdkman/bin/sdkman-init.sh"`
-   3. Verify the installation: `sdk version`
-2. Install _Coretto Java 8_, [link](https://sdkman.io/usage).
-   1. List the versions: `sdk list java`
-   2. Install the desired versions: `sdk install java 8.332.08.1-amzn`
-   3. If _JAVA_HOME_ is not set, re-source SDKMan's init script: `source "$HOME/.sdkman/bin/sdkman-init.sh"`
-3. Install the latest version of _Scala_: `sdk install scala`
-4. Install the latest version of _SBT_: `sdk install sbt`
+1. Install _Java 8_.
+2. Install _Scala_.
```

Comment:
SBT will download scala for you automatically, not required

Reply:
Removed the instruction to install Scala.